UnitTests/GitUI.Tests/Avatars/InitialsAvatarGeneratorTests.cs

```diff
@@ -10,36 +10,24 @@ namespace GitUITests.Avatars
     [TestFixture]
     public sealed class InitialsAvatarGeneratorTests
     {
-        [Test]
-        public void GetInitialsAndHashCode_return_initials_of_a_user()
+        [TestCase("albert.einstein@noreply.com", "albert einstein", "AE")]
+        [TestCase("albert.einstein@noreply.com", "albert z middlename einstein (k)", "AE")]
+        [TestCase("albert.einstein@noreply.com", "albert einstein (Nobel_price_winner)", "AE")]
+        [TestCase("albert.einstein@noreply.com", "EINSTEIN, Albert middlename (Nobel_price_winner)", "EM")]
```

Comment:
I'd expect the initials to be "EAM"... 🤔

Reply:
More than two looks weird and some international names have up to five parts, where to draw the limit? But it would still be something to identify them with. I chose esthetics over functionality.